config/application.yml

```diff
@@ -10,17 +10,21 @@
 #   stripe_api_key: sk_live_EeHnL644i6zo4Iyq4v1KdV9H
 #   stripe_publishable_key: pk_live_9lcthxpSIHbGwmdO941O1XVU
 
-SENDGRID_PASSWORD: "AQECAHjqALewp8JBJNxIQvR4oY795dyG7INaGR1glMsTEgetggAAAHQwcgYJKoZIhvcNAQcGoGUwYwIBADBeBgkqhkiG9w0BBwEwHgYJYIZIAWUDBAEuMBEEDGY/97q+tGJGGG5iNwIBEIAx6+8g3UugGcZqK0oszFJ5l9QC7PVeLQ5aFOyKJKNnxUZKPMWgwGju5l6auy0KnheViQ=="
-SENDGRID_USERNAME: "AQECAHjqALewp8JBJNxIQvR4oY795dyG7INaGR1glMsTEgetggAAAHQwcgYJKoZIhvcNAQcGoGUwYwIBADBeBgkqhkiG9w0BBwEwHgYJYIZIAWUDBAEuMBEEDD/C62QpqdcXGlEUxwIBEIAxVeOJrp9vOXRUAv0lh0jaAc7E+8eGFgnpZcOQbRJi7EtxQAJgmBYQHQ6tqEKKc4Hesg=="
-SECRET_KEY_BASE: "AQECAHjqALewp8JBJNxIQvR4oY795dyG7INaGR1glMsTEgetggAAAOMwgeAGCSqGSIb3DQEHBqCB0jCBzwIBADCByQYJKoZIhvcNAQcBMB4GCWCGSAFlAwQBLjARBAxjUy6F+DQ0NZ0NH/UCARCAgZv7/TEPBAow+srmtwcgFdEMbthnWmJVMZmR+NyMl0gaoqiQyYqJvAwbmgwDBYQKeVm1qSqvbKJchR86HQJ2icnUCH/OShiQs3H8Bo+ddnc97v/mhJx8MlR6fYyxZF79VmhYvfIaRO1HwTyCMjbV7dGbVw8kj0xZG7tXXyAubcdBMeJrnja5GghViu+I3VaZxQz1/RwbMhK5x49M7A=="
 
 <% require "aws_decrypt.rb" 
-  database_url = "postgresql://localhost/postgres"
-  #database_url = AwsDecrypt.decrypt_kms("AQECAHjqALewp8JBJNxIQvR4oY795dyG7INaGR1glMsTEgetggAAASwwggEoBgkqhkiG9w0BBwagggEZMIIBFQIBADCCAQ4GCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQM3apJFu88CTtQfpWdAgEQgIHg5FzviDrg2Qkl2HnO09j5KWORAq1CDtGaZtIATpvKoCHoxogGd4GyJ8M1/J/zHAtnapZiwe4AH22b3aUGQrQLDreu3t1kqp/4jVlkvz1Xa8+dCqeEeIsHgMXG1FTHrH3KZY27Fkg2mAHkJMlFI97oMoV8gj6O6UUfn/ZhNthrNkVSMZNZt2wFoZGXUC8s2rztJdnhQk1RusdDMp70+yoNwa470nOpXHNFoFHf1NVJ0Ih1HD8qSeViwWR7Q8sJSQCUYv9OPCGXkLNUWO1xNWY5d255dMrR/tSnEtugDNRXI1c=")
+    case ENV['RAILS_ENV']
+    when 'local'
```

Comment:
Indenting the lines after `when` is the convention.
The following is an example from 20% down this page: http://ruby-doc.org/docs/keywords/1.9/Object.html
```
case x
when 1,2,3
  puts "1, 2, or 3"
when 10
  puts "10"
else
  puts "Some other number"
end
```

This is what Rubocop would suggest:
```
    database_url = if ENV['RAILS_ENV'] == 'local'
                     "postgresql://localhost/postgres"
                   elsif ENV['RAILS_ENV'] ==  'development'
                     AwsDecrypt.decrypt_kms("AQECAHjqALewp8JBJNxIQvR4oY795dyG7INaGR1glMsTEgetggAAASwwggEoBgkqhkiG9w0BBwagggEZMIIBFQIBADCCAQ4GCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQM3apJFu88CTtQfpWdAgEQgIHg5FzviDrg2Qkl2HnO09j5KWORAq1CDtGaZtIATpvKoCHoxogGd4GyJ8M1/J/zHAtnapZiwe4AH22b3aUGQrQLDreu3t1kqp/4jVlkvz1Xa8+dCqeEeIsHgMXG1FTHrH3KZY27Fkg2mAHkJMlFI97oMoV8gj6O6UUfn/ZhNthrNkVSMZNZt2wFoZGXUC8s2rztJdnhQk1RusdDMp70+yoNwa470nOpXHNFoFHf1NVJ0Ih1HD8qSeViwWR7Q8sJSQCUYv9OPCGXkLNUWO1xNWY5d255dMrR/tSnEtugDNRXI1c=")  
                   elsif ENV['RAILS_ENV'] == 'qa'
                     AwsDecrypt.decrypt_kms("AQECAHh7ea2tyZ6phZgT4B9BDKwguhlFtRC6hgt+7HbmeFsrsgAAANMwgdAGCSqGSIb3DQEHBqCBwjCBvwIBADCBuQYJKoZIhvcNAQcBMB4GCWCGSAFlAwQBLjARBAx6tEcBuoN27fOPXIECARCAgYvNwgZyieISwY5tDHwk/HEW+V2wvFvL4wygxOZr860yRtMOaZeUAW9IYjOLbPA5T5kk6R4NoZMTYsvEsql1W11f9ITfbyKlJhCniyOGlB7TXuf3Qx+YjFvzFXoiTgrmSXXX37pmwJ8ZWFtjYVBzw5T3UdotLXPI3CIX+oBe/o+Bnp4hCx5N9m59UCEf")
                   end
```

Reply:
I'll fix the indentation.